tests/test_variants.py

```diff
@@ -12,8 +13,7 @@
 from conda_build import api, exceptions, variants
 from conda_build.utils import package_has_file
 
-thisdir = os.path.dirname(__file__)
-recipe_dir = os.path.join(thisdir, 'test-recipes', 'variants')
+from .utils import variants_dir
 
 
 def test_later_spec_priority(single_version, no_numpy_version):
```

Comment:
we could move this into the utils folder as well..

Reply:
move what?

Reply:
sorry wrong comment